doc/mpd.conf.5

```diff
@@ -108,6 +108,11 @@ for the format of this parameter.  Multiple audio_output sections may be
 specified.  If no audio_output section is specified, then MPD will scan for a
 usable audio output.
 .TP
+.B selective_44k_resample <yes or no>
```

Comment:
I'm not so happy with this option's name (and definition). There is no "44k" sample rate; it's "44.1k". And the documentation says it's not even about "44.1k" but about "44.1k or 48k".

Reply:
Which name would you suggest ?

Reply:
The problem is that everything about this is badly named, and it's hard to understand what this is all about. I mean, what is "selective resample" supposed to mean? And how can it be "based on 44k and 48k clocks"? I have no idea!
Once you rephrase the meaning and goal of this feature in a way normal people can understand easily, you can think about how to name the setting.